dali/base/dautils.cpp

```diff
@@ -908,7 +909,14 @@ StringBuffer &CDfsLogicalFileName::makeXPathLName(StringBuffer &lfnNodeName) con
                     c = toupper(*s);
                     // fall through
                 default:
-                    if (isalnum(c))
+                    if (first == s)
```

Comment:
Not sure it really matters, but can someone now create a file a::_49b whose lock would collide with the file a::1b ? And possibly also clashes between files that start with chr(210) and ones that start with chr(21) then the digit 1 ?